Makefile

```diff
@@ -94,9 +94,10 @@ uninstall-emacs:
 
 install-lib:
 	install -d $(LIB_DIR)
+	cp -r std $(LIB_DIR)
 	install -m 644 prelude.m31 $(LIB_DIR)/prelude.m31
 uninstall-lib:
-	rm -f $(LIB_DIR)/prelude.m31
+	rm -rf $(LIB_DIR)/* || true
```

Comment:
No star here, just:

    rm -f "$(LIB_DIR)"

and `|| true` might be unecessary.